xml/System.Text/Ascii.xml

```diff
@@ -16,8 +16,8 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Static class containing helper methods for working with ASCII encoded text as bytes or characters.</summary>
+    <remarks>For APIs which take both a source and a destination buffer, the behavior of the method is undefined if the source and destination buffers overlap, unless the API description specifies otherwise. The behavior of all APIs is undefined if another thread mutates the buffers while these APIs are operating on them. All case conversion APIs are culture-unaware.</remarks>
```

Comment:
```suggestion
    <remarks>For APIs that take both a source and a destination buffer, the behavior of the method is undefined if the source and destination buffers overlap, unless the description specifies otherwise. The behavior of all APIs is undefined if another thread mutates the buffers while these APIs are operating on them. All case-conversion APIs are culture-unaware.</remarks>
```